config/application.rb

```diff
@@ -291,6 +291,7 @@ def self.work_types
     # Settings in config/environments/* take precedence over those specified here.
     # Application configuration should go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded.
+    config.i18n.default_locale = :en
```

Comment:
In Rails 6 this setting is implied so this likely is not the issue afte rall 🤔

Reply:
given that, do you want to merge this or skip it?